.buildkite/pipeline.yml

```diff
@@ -341,6 +341,17 @@ steps:
       config: cpu
       queue: central
       slurm_ntasks: 1
+  
+  - label: ":computer: Rising Bubble hybrid invariant total energy"
+    key: "cpu_rising_bubble_hybrid_invariant"
```

Comment:
There seems to be a naming conflict with label in line# 335

Reply:
Conflicting key has been fixed.